core/services/wifi/WifiManager.py

```diff
@@ -160,13 +160,17 @@ async def remove_network(self, network_id: int) -> None:
         except Exception as error:
             raise ConnectionError("Failed to remove existing network.") from error
 
-    async def connect_to_network(self, network_id: int) -> None:
+    async def connect_to_network(self, network_id: int, force_connection: bool = True) -> None:
```

Comment:
I believe that a connect_to_work should always force the connection, so the force_connection variable is unnecessary

Reply:
Agree.

Reply:
Changed.